hypervisor/init_comm.go

```diff
@@ -79,6 +79,11 @@ func ReadVmMessage(conn *net.UnixConn) (*DecodedMessage, error) {
 		res = append(res, buf[:nr]...)
 		read = read + nr
 
+		// if condition satisified, message length may wrong , read again
+		if nr < want && read < needRead {
+			return ReadVmMessage(conn)
+		}
+
```

Comment:
That means I delete this tricky part and just pr the main lazy qemu code???And should I add the code that not closing all fds?